sway-core/src/semantic_analysis/namespace/module.rs

```diff
@@ -236,6 +236,7 @@ impl Module {
         src: &Path,
         dst: &Path,
         engines: &Engines,
+        is_absolute: bool,
```

Comment:
I think it should be clearer whether the source or destination path is the absolute one.

Reply:
Would using `CallPath` to bundle that information make sense here maybe?

Reply:
I renamed `is_absolute` to `is_src_absolute`.
Changing this src to CallPath should probably be done in a dedicated PR as it will likely propagate CallPaths to other places. I created an issue for this: #4700.

Reply:
Makes sense, I think this is good as is then.